### DISCUSSION TITLE
Command to compile nova assets?

### DISCUSSION THREAD

**Discussion by Gertiozuni** [laravel/nova-issues#4609] [Q&A] [answered]
Seeing that there some tailwind classes (ex. `w-1/3`, `w-2/3`, ...) not being available on published nova files, would  a command to recompile nova assets be useful?

Command might be using a base configuration that is published for the user to modify it as a preset ([Tailwind preset](https://tailwindcss.com/docs/presets)).

**Comment by davidhemphill** [comment, marked as answer]
Compiling Nova yourself for production purposes isn't supported, since it's fairly error-prone and easy to get out of sync with the latest changes. If you still really want to do it, you can follow the instructions from the docs: https://nova.laravel.com/docs/4.0/customization/frontend.html#vue-devtools
